Modules/Core/Common/CMake/itkGenerateNewCompilerFeatureDetectionHeaders.cmake

```diff
@@ -1,12 +1,20 @@
 ## Dynamically create compiler detection header
 include(WriteCompilerDetectionHeader)
+if (DEFINED ENV{ITK_ACCEPT_ALL_COMPILERS})
```

Comment:
This looks good.  Perhaps consider allowing both environmental variables or cmake variables to trigger the same behavior.
```
if (DEFINED ENV{ITK_ACCEPT_ALL_COMPILERS} OR ITK_ACCEPT_ALL_COMPILERS)
```

Reply:
That's a good idea. Done.